frontend/components/common/Modal.js

```diff
@@ -0,0 +1,125 @@
+import { useRef, useState, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import s from 'styled-components'
+
+import Shade from './Shade'
+import { Icon } from './Icon'
+import { LIGHT_GRAY } from '../../constants/colors'
+import {
+  BORDER_RADIUS_LG,
+  MD,
+  SM,
+  mediaMaxWidth,
+  LONG_ANIMATION_DURATION,
+} from '../../constants/measurements'
+import { fadeIn, fadeOut } from '../../constants/animations'
+
+const ModalWrapper = s.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  overflow-x: hidden;
+  overflow-y: auto;
+  z-index: 1002;
+  text-align: center;
+  animation-name: ${({ show }) => (show ? fadeIn : fadeOut)};
+  animation-duration: ${LONG_ANIMATION_DURATION};
```

Comment:
Just want to make sure that we don't need `ms` here—do the animations work locally for you?

Reply:
Added ms.